docs/transit-changes/README.md

```diff
@@ -155,32 +154,35 @@ User UI visualization of transit change detection results
 
 ##### Sequence
 
-![transit-changes-detection-logic](transit-changes-detection-logic.png)
+![transit_changes_detection_logic](transit_changes_detection_logic.png)
+- 0: **Detection task**  
+ ote.tasks.gtfs` is scheduled to run nightly or triggered manually from admin panel view.
 
 - 1: **Query from db services.**   
-The detection algorithm processes one transport service per time.  
-On this run those services are selected for change detection, which qualify for a new run. 
-(Reference: `services-for-nightly-change-detection`)
+The detection logic processes one service per iteration.  
+(Note: one service may include several interfaces, download logic is separate from this documentation)  
+Reference: `services-for-nightly-change-detection`
 - 2a: **Query route hashes from db.**  
-Fetch all routes for selected services. A route record includes information about route start and end times as well as unique hashes.  
+A route record contains route start and end times and unique hashes, into which details of traffic data is compressed for easy comparison.  
 Routes that have already ended are removed from further inspection.
-- 2b: **Override holidays in route hashes**  
-Holidays are earmarked by replacing day traffic hash using a keyword value. 
+- 2b: **Mark holiday/exception days in input data**  
+In list of days objects holidays are earmarked by replacing day traffic hash using a keyword value. 
 Replacing allows later steps to handle holidays.  
-Holiday dates are configured to NAP application itself.
-- 3b: **Find next changed week.**  
-Iterate weeks: compare 'current' week and a week in the future. Week is analyzed by comparing _day hashes_ of the same weekdays of the two separate weeks.  
-For efficiency, only simple day hash string comparison is done instead of detailed analysis of the daily routes and trips.  
-_Holiday days are skipped in analysis_ because they typically have different traffic, but such short changes are of no interest to user.  
+Holiday dates are read from db or as a fallback statically configured to NAP application.
```

Comment:
Should we tell that Holiday dates are fetched from RAE ?

Reply:
Added a reference into holidays chapter and replaced this part using a reference to that chapter.